apis/bigtable/v1alpha1/generate.sh

```diff
@@ -26,7 +26,8 @@ go run . generate-types \
   --api-version bigtable.cnrm.cloud.google.com/v1alpha1  \
   --resource BigtableAuthorizedView:AuthorizedView \
   --resource BigtableBackup:Backup \
-  --resource BigtableCluster:Cluster
+  --resource BigtableCluster:Cluster \
+  --resource BigtableSchemaBundle:SchemaBundle
```

Comment:
This did not generate any go types. I think we need to upgrade the git version for googleapis repo to pick up the "SchemaBundle" which was recently added. I'll do that.